src/operators/join.rs

```diff
@@ -485,16 +514,16 @@ impl<G, T1> JoinCore<G, T1::Key, T1::Val, T1::R> for Arranged<G,T1>
 /// The structure wraps cursors which allow us to play out join computation at whatever rate we like.
 /// This allows us to avoid producing and buffering massive amounts of data, without giving the timely
 /// dataflow system a chance to run operators that can consume and aggregate the data.
-struct Deferred<K, V1, V2, T, R1, R2, R3, C1, C2, M, D>
```

Comment:
the change here to not keep `mult` as a field seems like a unrelated refactor - is that right? or is there something more important going on?

Reply:
It was related, in that .. now that we do that pre-work before starting the operator, we would try and stash `Deferred` in a vector in two places, and their closures were not the same. They were syntactically identical, but Rust makes them two separate types. So, we couldn't bake the multiplication into the type if we wanted to populate `todo2` in two place.

Fall-out from the main change, rather than a core change itself.